web/src/transmission.js

```diff
@@ -31,8 +31,10 @@ export class Transmission extends EventTarget {
 
     // Initialize the helper classes
     this.action_manager = action_manager;
+    this.max_popups = 2;
```

Comment:
To ensure this value is not accidentally modified.

```js
static get max_popups() {
  return 2;
}
```

You can optionally define another property for convenience:

```js
static get default_popup_level() {
  return Transmission.max_popups - 1;
}
```